lib/quick_travel/scheduled_trip_product.rb

```diff
@@ -0,0 +1,28 @@
+require 'quick_travel/adapter'
+require 'quick_travel/base_product'
+
+module QuickTravel
+  class ScheduledTripProduct < BaseProduct
```

Comment:
Just a suggestion:

Instead of ``BaseProduct`` and ``ScheduledTripProduct`` use namespacing, i.e.: ``Product::Base`` and ``Product::ScheduledTrip``